src/petals/utils/packaging.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Dict, List, Tuple
+
+import torch
+from hivemind import nested_flatten, nested_pack
+
+# TODO: Move functions to hivemind
+
+
+def _mark_masked_tensor(index: int) -> bytes:
+    return b"__T" + str(index).encode()
+
+
+def _is_masked_tensor(item: Any) -> bool:
+    return isinstance(item, bytes) and item.startswith(b"__T") is not None
```

Comment:
```suggestion
    return isinstance(item, bytes) and item.startswith(b"__T")
```

Bug: it does return False/True